assets/component-image-with-text.css

```diff
@@ -311,13 +311,11 @@
 
 .image-with-text__heading {
   word-break: break-word;
-  hyphens: auto;
```

Comment:
> Remove usage of hyphens CSS property

Can you share a bit more context on the decision to remove hyphens? I probably missed this conversation, but would love to know the rationale behind this so we can have it documented :D